lib/fs/fs.go

```diff
@@ -127,6 +127,15 @@ func mustMkdirSync(path string) {
 // It doesn't remove the dir itself, so the dir may be mounted
 // to a separate partition.
 func RemoveDirContents(dir string) {
+	RemoveDirContentsWithOpt(dir, false)
+}
+
+// RemoveDirContentsWithOpt removes all the contents of the given dir if it exists.
+//
+// Flock file can be kept if parameter keepFlock is set true.
+// It doesn't remove the dir itself, so the dir may be mounted
+// to a separate partition.
+func RemoveDirContentsWithOpt(dir string, keepFlock bool) {
```

Comment:
It is better from readability PoV exporting `RemoveDirContentsKeepFlock()` func and making the `RemoveDirContentsWithOpt()` private, since it is more readable to have the following call:

```go
RemoveDirContentsKeepFlock(dir)
```
instead of
```go
RemoveDirContents(dir, true)
```

Reply:
Thanks for reviewing. I've fixed it.